osfoffline/filesystem_manager/osf_event_handler.py

```diff
@@ -46,71 +46,94 @@ def on_moved(self, event):
         :type event:
             :class:`DirMovedEvent` or :class:`FileMovedEvent`
         """
-
-        src_path = ProperPath(event.src_path, event.is_directory)
-        dest_path = ProperPath(event.dest_path, event.is_directory)
-
-        # determine and get what moved
-        if not self._already_exists(src_path):
-            try:
-                self._get_parent_item_from_path(src_path)
-            except ItemNotInDB:
-                # This means it was put into a place on the hierarchy being watched but otherwise not attached to a
-                # node, so it needs to be added just like a new event rather than as a move.
-
-                new_event = DirCreatedEvent(event.dest_path) if event.is_directory else FileCreatedEvent(event.dest_path)
-                yield from self._create_file_or_folder(new_event, src_path=dest_path)
+        try:
+            src_path = ProperPath(event.src_path, event.is_directory)
+            dest_path = ProperPath(event.dest_path, event.is_directory)
+        except Exception:
+            logging.exception('Exception caught: Invalid path specified')
+            AlertHandler.warn('Error moving {}. {} will not be synced.'.format('folder' if event.is_directory else 'file', src_path.split('/')[-1]))
```

Comment:
Also, in general, you probably want `os.path.basename(somestr)` here (rather than splitting and extracting the last item yourself)